<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-E7RVMER" ignored="0" name="Test" tests="4" failures="1" timestamp="2022-06-19T01:54:54 IDT" time="12.471" errors="0">
  <testcase name="tc_01_login" time="2.34" classname="saucedemo.com.RunTests.LoginTestNG"/>
  <testcase name="tc_02_login" time="2.258" classname="saucedemo.com.RunTests.LoginTestNG"/>
  <testcase name="tc_03_login" time="0.187" classname="saucedemo.com.RunTests.LoginTestNG">
    <failure type="java.lang.AssertionError" message="expected [Epic sadface: Password is incorrect] but found [Epic sadface: Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: expected [Epic sadface: Password is incorrect] but found [Epic sadface: Username and password do not match any user in this service]
at saucedemo.com.RunTests.LoginTestNG.tc_03_login(LoginTestNG.java:81)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- tc_03_login -->
  <testcase name="tc_04_login" time="2.27" classname="saucedemo.com.RunTests.LoginTestNG"/>
</testsuite> <!-- Test -->
